Side Bar.sublime-menu

```diff
@@ -0,0 +1,14 @@
+[
+    {
+        "caption": "Lsp: Rename File",
+        "mnemonic": "l",
+        "command": "lsp_rename_file",
+        "args": {"files": []}
+    },
+    {
+        "caption": "Lsp: Rename Folder",
```

Comment:
Can both be named the same to match the original menus?

Reply:
Can you add one more sentence on what you exactly expect?

Remove  "Lsp: Rename File" and  "Lsp: Rename Folder" and have only  "LSP: Rename..." in the sidebar context menu?

Reply:
Yes, that's what I mean.

Reply:
Implementation wise I was thinking to have two commands but with same title and with only one being visible at a time. That way you could still probably differentiate them based on some extra `args` and handle the logic correspondingly. If you can make it work with just one then that's fine too but that's not what I was thinking.

Reply:
> That way you could still probably differentiate them based on some extra args and handle the logic correspondingly.

I've spend quiet some time fiddling with TextInputHandler and I could not make it work. And while tacking this I wanted to address https://github.com/sublimelsp/LSP/pull/2498#issuecomment-2203180691 ... I tried a couple of approaches with TextInputHandler, and I could not address any of that TextInputHandler in a non hacky or hacky way. So I just switched to using `window.show_input_panel`

Reply:
If you find a nice approach, feel free to suggest or push changes directly.